parlai/core/torch_agent.py

```diff
@@ -522,6 +518,14 @@ def add_cmdline_args(cls, argparser):
             help='Decay factor for LR scheduler, or how much LR is multiplied by '
             'when it is lowered.',
         )
+        lr_group.add_argument(
```

Comment:
It might make sense to move the whole LR group parameters into the LRScheduler.add_cmdline_args, and then just call that from here.